doxia-site-renderer/src/test/java/org/apache/maven/doxia/siterenderer/DefaultSiteRendererTest.java

```diff
@@ -163,10 +163,12 @@ public void testRender()
 
         SiteRenderingContext ctxt = getSiteRenderingContext( decoration, "src/test/resources/site", false );
         ctxt.setRootDirectory( getTestFile( "" ) );
+        renderer.copyResources( ctxt, getTestFile( OUTPUT ) );
```

Comment:
Why is this necessary now? I can see you have moved the resources. Won't the site generation fail?

Reply:
I've racked my brain and I don't remember why the change was necessary, however the site generation worked for me - I cheated and updating the site plugin in the poms to the not-yet-existing-until-the-next-patch-is-approved v4.0-SNAPSHOT of the maven-site-plugin, and the site generation worked fine.

Reply:
Lets put it differently, does not work if you don't make this copy? Because only only change style of the Velocity template at the end...

Reply:
Ran the test suite with these lines commented out, and got this:

    SEVERE: Exception loading file:/Users/minfrin/src/apache/sandbox/maven/maven-doxia-sitetools/doxia-site-renderer/target/output/css/maven-base.css
    com.gargoylesoftware.htmlunit.FailingHttpStatusCodeException: 404 Not Found for file:/Users/minfrin/src/apache/sandbox/maven/maven-doxia-sitetools/doxia-site-renderer/target/output/css/maven-base.css

The test suite still passes though - is this exception present in the original code?

Reply:
I will check

Reply:
Here is the issue: https://builds.apache.org/job/maven-box/job/maven-doxia-sitetools/job/htmlunit-2.24/2/testReport/junit/org.apache.maven.doxia.siterenderer/DefaultSiteRendererTest/windows_jdk11___Build_windows_jdk11___testRender/

It needs to be fixed first with DOXIASITETOOLS-198